lang/rapidjson/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem                      1.0
+PortGroup                       github 1.0
+PortGroup                       cmake 1.1
+
+github.setup                    Tencent rapidjson b32cd9421c5e3cbe183a99b6537ce11441e50656
+version                         20180424
+license                         MIT
+categories                      lang devel
+platforms                       darwin
+maintainers                     @Lord-Kamina
```

Comment:
Should also [include an email address](https://guide.macports.org/#reference.keywords.maintainers), so you can be contacted from other systems by email (for example our buildbot).

Reply:
I thought it was one or the other, I guess I should correct the other PRs as well.